src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/TestDependenciesRestClient.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.starter.addon.microprofile.servers;
+
+import org.eclipse.microprofile.starter.addon.microprofile.servers.model.SupportedServer;
+import org.eclipse.microprofile.starter.core.model.JessieMaven;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestDependenciesRestClient {
+
+    private static TestDependenciesRestClient INSTANCE = new TestDependenciesRestClient();
+
+    private Map<SupportedServer, JessieMavenWithVersion> serverSpecificData;
+
+    private TestDependenciesRestClient() {
+        serverSpecificData = new HashMap<>();
+        serverSpecificData.put(SupportedServer.PAYARA_MICRO, new JessieMavenWithVersion("org.glassfish.jersey.core", "jersey-client", "2.25.1"));
```

Comment:
I am wondering whether the versions can be injected from the value defined in pom.xml.  I know in Liberty, the variables can be specified under bootstrapping property element in pom.xml? Do you know whether other app server has the similar support.

Reply:
@Emily-Jiang I don't think this will be possible.

- Not all implementations are using the Rest Client, most are just using the server artifact. (which can have a different version)
- We need then to download and parse the pom, possible with transitive dependencies which makes it difficult.

And it doesn't really matter, we use it just to send a certain request and it can be in any technology/framework/... (Spring, nodeJS, .NET, ...)

Reply:
We need to document about version dependency matching.

Reply:
I have added some more information about the usage of the JAX-RS rest client in the generated readme.md and JWTClient.java files. See https://github.com/eclipse/microprofile-starter/pull/100/commits/52f380bbe22b814849906beea54b1fe7cfc36f4f